specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/jobDetails"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total records count number."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "jobDetails": {
+      "description": "Job details.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The job id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The job name. Is not required for the name to be unique and it's only used for display purposes."
+        },
+        "containerUri": {
+          "type": "string",
+          "description": "The blob container SAS uri, the container is used to host job data."
+        },
+        "inputDataUri": {
+          "type": "string",
+          "description": "The input blob SAS uri, if specified, it will override the default input blob in the container."
+        },
+        "inputDataFormat": {
+          "type": "string",
+          "description": "The format of the input data."
+        },
+        "inputParams": {
+          "type": "object",
+          "description": "The input parameters for the job. JSON object used by the target solver. It is expected that the size of this object is small and only used to specify parameters for the execution target, not the input data."
+        },
+        "providerId": {
+          "type": "string",
+          "description": "The unique identifier for the provider."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target identifier to run the job."
+        },
+        "metadata": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The job metadata. Metadata provides client the ability to store client-specific information"
+        },
+        "outputDataUri": {
+          "type": "string",
+          "description": "The output blob SAS uri. When a job finishes successfully, results will be uploaded to this blob."
+        },
+        "outputDataFormat": {
+          "type": "string",
+          "description": "The format of the output data."
+        },
+        "status": {
+          "type": "string",
+          "description": "The job status.",
+          "readOnly": true,
+          "enum": [
+            "Waiting",
+            "Executing",
+            "Succeeded",
+            "Failed",
+            "Cancelled"
+          ],
+          "x-ms-enum" : {
+            "name": "JobStatus",
+            "modelAsString": true
+          }
+        },
+        "creationTime": {
+          "type": "string",
```

Comment:
"format": "date-time"

Reply:
This applies to all properties that are date-times. Also, please make sure that you keep precision to millisecond (three decimals).